x/delegation/client/cli/tx_delegation_native.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"errors"
+
+	sdkmath "cosmossdk.io/math"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/ExocoreNetwork/exocore/x/delegation/types"
+)
+
+func CmdDelagateNativeToken() *cobra.Command {
+	cmd := &cobra.Command{
+		// TODO: only support native token for now
+		Use:   "delegate-native operator amount",
+		Short: "Broadcast message delegate-native to delegate native token",
```

Comment:
A bit repetitive, please change
```suggestion
		Short: "Broadcast a transaction to delegate amount of native token to the operator",
```